changedetectionio/templates/edit.html

```diff
@@ -287,9 +287,9 @@ <h2 >Click here to Start</h2>
                                 {% endif %}
                             </ul>
                         </li>
-                        <li>XPath - Limit text to this XPath rule, simply start with a forward-slash,
+                        <li>XPath - Limit text to this XPath rule, simply start with a forward-slash. To specify XPath to be used explicitly or the XPath rule starts with an XPath function: Prefix with <code>xpath:</code>
                             <ul>
-                                <li>Example:  <code>//*[contains(@class, 'sametext')]</code> or <code>xpath://*[contains(@class, 'sametext')]</code>, <a
+                                <li>Example:  <code>//*[contains(@class, 'sametext')]</code> or <code>xpath:count(//*[contains(@class, 'sametext')])</code>, <a
```

Comment:
XPath script can start with just a function. However, XPath's built-in functions are more than 200. That is why I suggest specifying `xpath:`. In your script, if it doesn't start with `/` or `xpath:`, then it would be assumed CSS selector, maybe.